cmd/acr/purge.go

```diff
@@ -568,38 +586,48 @@ func dryRunPurge(ctx context.Context, acrClient api.AcrCLIClientInterface, login
 			}
 			return -1, -1, err
 		}
-		// This will act as a set if a key is present then it should not be deleted because it is referenced by a multiarch manifest
+		// This will act as a set if a key is present then it should not be deleted
```

Comment:
Nit. Add some clarifying punctuation:
	// This map will act as a set. If a key is present, then the corresponding manifest should not
        // be deleted because it is referenced by a multiarch manifest or otherwise references a subject

Reply:
done. good point to enhance the readability.